src/lib/features/like-episode/model/like.ts

```diff
@@ -30,19 +30,28 @@ export const likesStore = persistentWritable(new Set<number>(), {
 });
 
 async function addCloudLike(episodeId: number) {
-  if (!get(user)) return;
-  await supabaseClient.from('favourites').insert([
-    {
-      // eslint-disable-next-line camelcase
-      episode_id: episodeId,
-    },
-  ]);
+  try {
+    await supabaseClient.from('favourites').insert([
+      {
+        // eslint-disable-next-line camelcase
+        episode_id: episodeId,
+      },
+    ]);
+  } catch (e) {
+    console.error(e);
```

Comment:
issue: I would appreciate if we kept on hiding stack traces, continuing on the work I've done in https://github.com/aabounegm/cast/pull/171.

suggestion: here I would suggest propagating the error to the caller to let them handle it however they see fit